docs/user/manual.adoc

```diff
@@ -186,18 +186,7 @@ $ pacman -S rust-analyzer
 
 ==== Gentoo Linux
 
-`rust-analyzer` is available in the GURU repository:
-
-- https://gitweb.gentoo.org/repo/proj/guru.git/tree/dev-util/rust-analyzer?id=9895cea62602cfe599bd48e0fb02127411ca6e81[`dev-util/rust-analyzer`] builds from source
-- https://gitweb.gentoo.org/repo/proj/guru.git/tree/dev-util/rust-analyzer-bin?id=9895cea62602cfe599bd48e0fb02127411ca6e81[`dev-util/rust-analyzer-bin`] installs an official binary release
-
-If not already, GURU must be enabled (e.g. using `app-eselect/eselect-repository`) and sync'd before running `emerge`:
-
-[source,bash]
-----
-$ eselect repository enable guru && emaint sync -r guru
-$ emerge rust-analyzer-bin
-----
+`rust-analyzer` is installed when the `rust-analyzer` use flag is set for dev-lang/rust or dev-lang/rust-bin. You also need to set the `rust-src` use flag.
```

Comment:
```suggestion
`rust-analyzer` is installed when the `rust-analyzer` use flag is set for `dev-lang/rust` or `dev-lang/rust-bin`.
You also need to set the `rust-src` use flag.

This is similar to using the `rustup` component, so it will install a version that lags behind the GitHub releases.
```

Reply:
I'm happy with your suggested changes @lnicola  - thanks. If a gentoo user wants to use the latest rust-analyzer version, they will need to make their own ebuild (in a local overlay) to install the latest release. I think the removal from the guru repository was perhaps not the best decision, given the different version situation, so I will ask whether it is worth re-instating the ebuild there, or even in the main gentoo repository.